mongo_connector/oplog_manager.py

```diff
@@ -644,10 +730,23 @@ def do_dump(dm, error_queue):
     def _get_oplog_timestamp(self, newest_entry):
         """Return the timestamp of the latest or earliest entry in the oplog.
         """
+        query = {'op': {'$ne': 'n'}}
+        if self.oplog_ns_set and not self.oplog_ex_ns_set:
+            query['ns'] = {'$in': self.oplog_ns_set}
+
+        if self.oplog_ex_ns_set and not self.oplog_ns_set:
+            query['ns'] = {'$nin': self.oplog_ex_ns_set}
+
+        if self.oplog_ns_set and self.oplog_ex_ns_set:
+            query = {'$and': [{'ns': {'$nin': self.oplog_ex_ns_set}},
+                              {'ns': {'$in': self.oplog_ns_set}}]}
+
+        LOG.debug("Cursor query: %s" % query)
+
```

Comment:
`_get_oplog_timestamp` no longer filters by namespace. I think you can simply remove these lines. Then after addressing my other comment the tests should pass again.